packages/vx-geo/src/projections/Projection.js

```diff
@@ -47,14 +49,16 @@ export default function Projection({
   if (clipExtent) currProjection.clipExtent(clipExtent);
   if (scale) currProjection.scale(scale);
   if (translate) currProjection.translate(translate);
-  if (center) currProjection.translate(center);
+  if (center) currProjection.center(center);
   if (rotate) currProjection.rotate(rotate);
   if (precision) currProjection.rotate(precision);
   if (fitExtent) currProjection.fitExtent(...fitExtent);
   if (fitSize) currProjection.fitSize(...fitSize);
 
   const path = geoPath().projection(currProjection);
 
+  if (pointRadius) path.pointRadius(pointRadius);
```

Comment:
```suggestion
  if (Number.isFinite(pointRadius)) path.pointRadius(pointRadius);
```

👋 folks! Do you if `pointRaidus` can be `0`? Would it be a valid input? I can change this in https://github.com/airbnb/visx/pull/1767 if that makes sense.

Reply:
hey @kachkaev  according to the `d3-geo` docs this should be a 2-element numerical array https://d3js.org/d3-geo/projection#projection_center

that's consistent with our `@visx/geo` types / docs https://airbnb.io/visx/docs/geo#Projection_center

so I don't think `0` is valid.

Reply:
I think it’s related to https://d3js.org/d3-geo/path#path_pointRadius (one number)

Reply:
🤦 🤦 🤦  sorry no idea how I misread that one. 100% agree with you/thanks for adding in your new PR 🙏